<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="5" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-06-24T11:09:21 EDT" name="Functional" finished-at="2020-06-24T11:10:27 EDT" duration-ms="66026">
    <groups>
    </groups>
    <test started-at="2020-06-24T11:09:21 EDT" name="Base" finished-at="2020-06-24T11:09:39 EDT" duration-ms="17687">
      <class name="expediatest.BaseExpedia">
        <test-method signature="expediahome()[pri:0, instance:expediatest.BaseExpedia@a514af7]" started-at="2020-06-24T11:09:21 EDT" name="expediahome" finished-at="2020-06-24T11:09:39 EDT" duration-ms="17567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- expediahome -->
      </class> <!-- expediatest.BaseExpedia -->
    </test> <!-- Base -->
    <test started-at="2020-06-24T11:09:39 EDT" name="CheckBox" finished-at="2020-06-24T11:10:09 EDT" duration-ms="30138">
      <class name="expediatest.CheckBox">
        <test-method signature="expediahome()[pri:0, instance:expediatest.CheckBox@8f4ea7c]" started-at="2020-06-24T11:09:39 EDT" name="expediahome" finished-at="2020-06-24T11:10:00 EDT" duration-ms="20809" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- expediahome -->
        <test-method signature="box()[pri:0, instance:expediatest.CheckBox@8f4ea7c]" started-at="2020-06-24T11:10:00 EDT" name="box" finished-at="2020-06-24T11:10:09 EDT" duration-ms="9286" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- box -->
      </class> <!-- expediatest.CheckBox -->
    </test> <!-- CheckBox -->
    <test started-at="2020-06-24T11:10:09 EDT" name="Assertions" finished-at="2020-06-24T11:10:09 EDT" duration-ms="31">
      <class name="expediatest.Assertions">
        <test-method signature="AssertionsRep()[pri:0, instance:expediatest.Assertions@57a3af25]" started-at="2020-06-24T11:10:09 EDT" name="AssertionsRep" finished-at="2020-06-24T11:10:09 EDT" duration-ms="15" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Yahoo] but found [gmail],
expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Yahoo] but found [gmail],
expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at expediatest.Assertions.AssertionsRep(Assertions.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AssertionsRep -->
      </class> <!-- expediatest.Assertions -->
    </test> <!-- Assertions -->
    <test started-at="2020-06-24T11:10:09 EDT" name="Test4" finished-at="2020-06-24T11:10:27 EDT" duration-ms="17809">
      <class name="expediatest.expediaElement">
        <test-method signature="expediahome()[pri:0, instance:expediatest.expediaElement@223191a6]" started-at="2020-06-24T11:10:09 EDT" name="expediahome" finished-at="2020-06-24T11:10:22 EDT" duration-ms="13018" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- expediahome -->
        <test-method signature="exElement()[pri:0, instance:expediatest.expediaElement@223191a6]" started-at="2020-06-24T11:10:22 EDT" name="exElement" finished-at="2020-06-24T11:10:27 EDT" duration-ms="4786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exElement -->
      </class> <!-- expediatest.expediaElement -->
    </test> <!-- Test4 -->
  </suite> <!-- Functional -->
</testng-results>
